tests/common/fixtures/conn_graph_facts.py

```diff
@@ -1,12 +1,26 @@
 import pytest
 import os
+import json
+
+from ansible_host import AnsibleHost
 
 @pytest.fixture(scope="module")
-def conn_graph_facts(testbed_devices):
+def conn_graph_facts(testbed_devices, ansible_adhoc):
     dut = testbed_devices["dut"]
     localhost = testbed_devices["localhost"]
 
     base_path = os.path.dirname(os.path.realpath(__file__))
-    conn_graph_file = os.path.join(base_path, "../../../ansible/files/lab_connection_graph.xml")
-    conn_graph_facts = localhost.conn_graph_facts(host=dut.hostname, filename=conn_graph_file)['ansible_facts']
+    # default lab graph file
+    lab_conn_graph_file = os.path.join(base_path, "../../../ansible/files/lab_connection_graph.xml")
+    # json file contains mapping from inventory file name to its corresponding graph file
+    inv_mapping_file = os.path.join(base_path, "../../../ansible/group_vars/all/inv_mapping.json")
```

Comment:
what is the inv mapping file? I cannot find it here. https://github.com/Azure/sonic-mgmt/tree/master/ansible/group_vars/all

Reply:
The default is still 'lab_connection_graph.xml'. if we want to use any other graph, then we need to specify the inv mapping file. We will use it internally to pick up our specific inventory graph

Reply:
Using the inventory mapping json in all cases now